sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/NuGetMetadataReference.cs

```diff
@@ -39,6 +39,9 @@ internal static class NuGetMetadataReference
             NugetMetadataFactory.Create("xunit", "1.9.1")
             .Concat(NugetMetadataFactory.Create("xunit.extensions", "1.9.1"));
 
+        public static IEnumerable<MetadataReference> NETStandardV2_1_0 =>
```

Comment:
I guess this should rather be in `FrameworkMetadataReference` - closer to `Netstandard` which gets the default one installed

Some alternatives:

- If you add a reference to netstandard 2.1 in the unit test csproj and re-use netstandard, will it work? (like we have it in the [security C# FE](https://github.com/SonarSource/sonar-security/blob/master/frontend/csharp/src/SonarAnalyzer.Security.Test/SonarAnalyzer.Security.Test.csproj#L13) UTs) - if it works and doesn't break existing tests, we should just upgrade...

- if the above is breaking tests (change of API between the previous netstandard maybe 1.x and the new one), then is it possible to reuse the same pattern from the [FrameworMetadataReference](https://github.com/SonarSource/sonar-dotnet/blob/cz/S5542/sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/FrameworkMetadataReference.cs#L36) and basically use Roslyn API?